### PR TITLE
Use syft version v0.15.1 in Dockerfile

### PR DESCRIPTION
**Pull request by dakaneye**

### CODE REVIEW

**Review by luhring — APPROVED** [review]
LGTM! Although Black is mad at you 💢

**Comment by dspalmer99** [comment]
LGTM, pending successful CI run.

**Review by dspalmer99 — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -10,6 +10,6 @@ passenv=ANCHORE_* DOCKER_* NO_PROXY
 
 commands=
     coverage erase
-    coverage run -m pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs}
+    coverage run -m pytest -vv --junitxml=.tox/results/tox/junit-{envname}.xml {posargs}
```
Comment (dspalmer99) [inline tox.ini:13]: @dakaneye Does this affect/break any of the existing, unrelated CI test machinery?

**Review by zhill — APPROVED** [review]
lgtm

File: tox.ini (hunk 1 above)
Comment (dakaneye) [reply tox.ini:13]: Just makes it easier to troubleshoot test failures by increasing verbosity

Comment (dspalmer99) [reply tox.ini:13]: Nice, thanks

**Pull request merged**